### ISSUE TITLE
[Backend] Adding various security features

### ISSUE THREAD

**Issue by udaymittal7** [HITK-TECH-Community/Community-Website#517]
Setting various HTTP headers using helmet
Data sanitization against no SQL injection and XSS
Rate limiter so the server won't get overload
Preventing parameter pollution

If you find it relevant, please assign me.
I am a participant of GSSOC'21 so please add the relevant GSSOC tags too.

**Comment by jackfrost13** [comment]
Can you please elaborate on it a bit? 
I don't feel the need for a rate limiter for the project.

**Comment by jackfrost13** [comment]
okay, you can take this is just as an additional check but we don't feel the need for these explicitly at the moment but yeah these are somewhat good to have

**Comment by jackfrost13** [comment]
@udaymittal7 Please update progess on this

**Comment by Kriti-bit** [comment]
Hi, I am a GSSoC'21 Participant. I would like to work on this issue, please assign it to me.

**Comment by himanshusanecha** [comment]
I am interested to work upon this.

**Comment by AyushSingh22** [comment]
@himanshusanecha  Go ahead with this and kindly update here after updating.

**Comment by himanshusanecha** [comment]
yes sure, I am starting to work on this issue. Sorry for the delay due to exams
